### PR TITLE
feat: create setting for semi private mode with only percentage shown

### LINKED ISSUE #2901: "Semi" Privacy Mode with only Percentage %
## Abstract
Had an idea for an "Semi" Privacy Mode which only shows your assets with percentages, instead of nothing as the current privacy mode.

## Motivation
For sharing with someone some asset information, it would be helpful to show a semi privacy mode with percentages.
So you can show them the correct portfolio but not the amount of your Tokens
Nothing important but maybe helpful and interesting for one or another.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #2901
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Create setting for semi private mode with only percentage shown

### CODE REVIEW

**Comment by yabirgb** [comment]
It looks great!

File: frontend/app/src/components/UserDropdown.vue [hunk 1]
```diff
@@ -106,3 +161,129 @@ export default class UserDropdown extends Vue {
   }
 }
 </script>
+
+<style scoped lang="scss">
+/* stylelint-disable */
```
Comment (kelsos) [inline frontend/app/src/components/UserDropdown.vue:166]: Disabling a few specific rules case by case is ok but I would advise against disabling all the style rules.

Comment (lukicenturi) [reply frontend/app/src/components/UserDropdown.vue]: I already move the disabled comment to only specific rule

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**